app/models/user.rb

```diff
@@ -2,4 +2,8 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments
   has_many :likes
+
+  def last_three_posts
+    Post.last(3)
+  end
```

Comment:
- [ ] Same here. It will show you `3 last post` but not in descending order. For that, you have to include one more condition.

Hint: `posts.order('created_at Desc').limit(3)`

Reply:
Same question here, please. `posts` or `Post`?

Reply:
Hi @HENRYKC24,

- Same here. Rails recommend using `posts` instead of class name here.

Reply:
:+1:

Reply:
This issue is well resolved! ✔️